pkg/registry/registry.go

```diff
@@ -98,6 +99,59 @@ func (reg *Registry) setupRoutes() {
 	})
 }
 
+// Request logger for Chi using Zap as the logger.
+func (reg *Registry) RequestLogger() func(next http.Handler) http.Handler {
+	// This method is _adapted_ from https://github.com/moul/chizap/blob/0ebf11a6a5535e3c6bb26f1236b2833ae7825675/chizap.go
+	// written by Manfred Touron, licensed under the MIT license.
+	//
+	//
+	// Copyright (c) 2021  Manfred Touron <oss@moul.io> (manfred.life)
```

Comment:
Overkill to copy the entire license into the file :) It's fine to just link to the code with an external link and inlcude the SPDX identifier.

Reply:
Reworked!